libs/wingc/src/ast.rs

```diff
@@ -423,6 +424,12 @@ pub struct Interface {
 #[derive(Debug)]
 pub enum BringSource {
 	BuiltinModule(Symbol),
+	/// A Wing library (containing Wing source files).
+	/// root_file is a relative path to the root file of the library (relative to the compilation entrypoint)
+	WingModule {
+		name: Symbol,
+		root_file: PathBuf,
+	},
 	JsiiModule(Symbol),
 	WingFile(Symbol),
```

Comment:
I believe this should have been `WingModule`, no?

Reply:
Good call, I'll rename 👍